src/main/java/com/github/vaerys/commands/general/GetReminders.java

```diff
@@ -0,0 +1,101 @@
+package com.github.vaerys.commands.general;
+
+import com.github.vaerys.enums.ChannelSetting;
+import com.github.vaerys.enums.SAILType;
+import com.github.vaerys.main.Globals;
+import com.github.vaerys.main.Utility;
+import com.github.vaerys.masterobjects.CommandObject;
+import com.github.vaerys.objects.ReminderObject;
+import com.github.vaerys.templates.Command;
+import com.github.vaerys.utilobjects.XEmbedBuilder;
+import sx.blah.discord.handle.obj.IChannel;
+import sx.blah.discord.handle.obj.IGuild;
+import sx.blah.discord.handle.obj.Permissions;
+
+import java.time.Instant;
+import java.util.List;
+
+public class GetReminders extends Command {
+
+    @Override
+    public String execute(String args, CommandObject command) {
+        List<ReminderObject> reminders = Globals.getGlobalData().getRemindersUser(command.user.longID);
+
+        if (reminders.size() == 0) return "> You have no reminders set.";
+
+        XEmbedBuilder embed = new XEmbedBuilder(command);
+        int i = 1;
+        for (ReminderObject r : reminders) {
+            long timeSecs = r.getExecuteTime() - Instant.now().getEpochSecond();
+            String timeFormatted = Utility.formatTime(timeSecs, true);
+
+            IChannel channel = command.client.get().getChannelByID(r.getChannelID());
+            if (channel == null) {
+                // try to verify it's a DM channel instead.
+                channel = command.user.getDmChannel();
+                if ( channel == null || (channel.getLongID() != r.getChannelID()) ) continue;
```

Comment:
the channel check will break the system. `(channel.getLongID() != r.getChannelID())` needs to be removed.